src/main.rs

```diff
@@ -75,6 +75,7 @@ fn main() {
         output_path: args.output_path,
         optimize_egglog: args.optimize_egglog,
         optimize_brilift: args.optimize_brilift,
+        optimize_brillvm: args.optimize_brilift,
```

Comment:
bug here? is the brillvm flag supposed to be different?

Reply:
Yes, definitely a bug, nice catch